src/javascript/app_2/Stores/Modules/Contract/contract_store.js

```diff
@@ -111,7 +119,7 @@ export default class ContractStore extends BaseStore {
     @action.bound
     handleSell(response) {
         if (response.error) {
-            this.sell_info = {
+            this.sell_info         = {
```

Comment:
@easteregg 
it doesn't need to add extra space when the statement is not settled in the one line.

Reply:
I'll fix my web storm code style formatter.